server/fleet/capabilities.go

```diff
@@ -65,6 +65,7 @@ var ServerOrbitCapabilities = CapabilityMap{
 // ServerDeviceCapabilities is a set of capabilities that server-side,
 // Device-related endpoint supports.
 // **it shouldn't be modified at runtime**
+// TODO(mna): should this be updated to indicate the MDM manual enrollment profile support?
```

Comment:
My (limited) understanding of Fleet Desktop capabilities is that this is only used for API calls that Fleet Desktop itself makes. In the case of this PR, the MDM manual enrollment profile is called by the "My Device" _page_ which is outside fleet desktop and under the control of (rendered by) the Fleet Server, so I don't think it needs to be added to the capabilities map.

Reply:
indeed, what you described makes sense 👍